scripts/tests/api_compare/docker-compose.yml

```diff
@@ -2,70 +2,179 @@
 version: "3.8"
 
 services:
-  # Initialisation service. Downloads the shared assets to facilitate the setup of the other services.
-  init:
-    image: ${FOREST_IMAGE}
-    user: root
+  init: 
+    image: ubuntu:focal
     volumes:
       - node-data:/data
-      - filecoin-proofs:/proofs
-    env_file:
-      - .env
-    entrypoint: ["/bin/bash", "-c" ]
+    networks:
+      - api-tests
+    environment:
+      - FIL_PROOFS_PARAMETER_CACHE=${FIL_PROOFS_PARAMETER_CACHE}
+    entrypoint: [ "/bin/bash", "-c" ]
     command:
       - |
         set -euxo pipefail
-        # if there are some files in the data directory, then we don't need to fetch the snapshot
-        if [ "$$(ls -A /data/snapshot/)" ]; then
-          echo "Snapshot already fetched"
-        else
-          mkdir -p /data/snapshot
-          forest-tool snapshot fetch  --chain ${CHAIN} --directory /data/snapshot/
-        fi
-        # copy proofs to the shared volume, if they are not already there
-        if [ "$$(ls -A /proofs/)" ]; then
-          cp -r ${FIL_PROOFS_PARAMETER_CACHE}/* /proofs/
-        fi
+        apt-get update
+        apt-get install -y aria2
+        aria2c -d /data -x5 https://forest-archive.chainsafe.dev/latest/calibnet/
```

Comment:
@LesnyRumcajs 

> It'd be better to have a docker image with aria

Done.

> I suggest we check if there's a snapshot already in the volume to avoid re-downloading it.

`aria2c` can automatically avoid downloading the same snapshot.

Reply:
> aria2c can automatically avoid downloading the same snapshot.

Yeah, but it's not always the same file. For example, it will resolve and download `forest_snapshot_calibnet_2024-03-26_height_1471334.forest.car.zst`, but a few minutes later it will grab `forest_snapshot_calibnet_2024-03-26_height_1471394.forest.car.zst`.

Reply:
Yes, that happens only locally, if a snapshot is very old, it takes a long time for forest and lotus to sync. I feel it makes sense to download the latest if user did not manually cleanup the volume that is a few days old. Does that make sense?

Reply:
It's not very old, we are producing calibnet snapshots at a frequency of around 1 snapshot per hour. That's why I introduced the logic there. https://forest-archive.chainsafe.dev/list/calibnet/latest

Reply:
OK, added it back